lib/take_away.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+class Takeaway
+
+  attr_reader :my_order, :check_out
+  
+  MENU = { "hotdog"=>1.99,
+          "cheesebuger"=>1.49,
+          "Hamburger"=>0.99,
+          "Fries"=>0.89, }
+
+  def initialize(menu_klass = Menu.new)
```

Comment:
`_klass` convention refers to an actual class, here you are passing an instance, not a class. It should just be `menu: Menu.new`